lang/java/reef-runtime-standalone/src/main/java/org/apache/reef/runtime/standalone/client/StandaloneRuntimeConfiguration.java

```diff
@@ -18,21 +18,37 @@
  */
 package org.apache.reef.runtime.standalone.client;
 
-import org.apache.reef.annotations.Unstable;
+import org.apache.reef.runtime.local.LocalClasspathProvider;
+import org.apache.reef.runtime.local.client.LocalJobSubmissionHandler;
+import org.apache.reef.runtime.local.client.ExecutorServiceConstructor;
 import org.apache.reef.client.parameters.DriverConfigurationProviders;
 import org.apache.reef.runtime.common.client.CommonRuntimeConfiguration;
+import org.apache.reef.runtime.common.client.DriverConfigurationProvider;
+import org.apache.reef.runtime.common.client.api.JobSubmissionHandler;
+import org.apache.reef.runtime.common.files.RuntimeClasspathProvider;
+import org.apache.reef.runtime.standalone.client.parameters.NodeFolder;
 import org.apache.reef.runtime.standalone.client.parameters.NodeListFilePath;
-import org.apache.reef.runtime.standalone.client.parameters.RootFolder;
+import org.apache.reef.runtime.local.client.parameters.RootFolder;
 import org.apache.reef.tang.ConfigurationProvider;
 import org.apache.reef.tang.formats.*;
 
+import java.util.concurrent.ExecutorService;
+
 /**
  * A ConfigurationModule to configure the standalone resourcemanager.
  */
-@Unstable
 public final class StandaloneRuntimeConfiguration extends ConfigurationModuleBuilder {
 
   /**
+   * The file with information of remote nodes.
```

Comment:
Could you add to the comment that the standalone runtime assume the Driver to run on the same node as Client, and thus a local file path is expected?